chcemvediet/tests/__init__.py

```diff
@@ -135,15 +136,17 @@ def _create_obligee(self, **kwargs):
                 })
 
     def _create_obligee_tag(self, **kwargs):
+        tag = u'{:03d}'.format(self.counter.next())
         return self._call_with_defaults(ObligeeTag.objects.create, kwargs, {
-                u'key': u'Default Testing Key',
-                u'name': u'Default Testing Name',
+                u'key': u'Default Testing Key {}'.format(tag),
+                u'name': u'Default Testing Name {}'.format(tag),
                 })
 
     def _create_obligee_group(self, **kwargs):
+        tag = u'{:03d}'.format(self.counter.next())
```

Comment:
Nebolaj to `tag` nech sa to nemyli s obligee tagom. Skor niecko ako "suffix" alebo tak nejak. Rovnako na ostatnych miestach.

Reply:
@martinmacko47 Radsej by som to nazval "serno" (serial number), alebo "key", "serial key", lebo to odkazuje na to, ze to ma byt identifikator.
Nazov premennej `tag` je pouzity aj v metode `_create_user`. Mam to zmenit aj tam? Tu nedochadza k zamene s `ObligeeTag`. Ak ano, podlam mna by pasoval aj nazov "pk", pretoze by malo platit (aspon v testoch) ze djangom novo vygenerovany primarny kluc pre Usera bude o 1 vacsi ako posledna instancia. tj. `new_user.pk == User.objects.latest(u'pk').pk + 1`

Reply:
@viliambalaz Urcite nie `pk`, pretoze `self.counter` rastie rychlejsie nez `pk`, kedze pouzivame jeden counter na vsetky modely.

"serno" znie divne, co tak napriklad `number`? alebo len `nr`?

A ano, prerob to aj v `_create_user`. Ked sa pisal `_create_user`, tak obligee tag este neexistoval, tak sa to nemylilo. Teraz je lepsie to aj tam premenovat. A tiez to tam prerob, nech sa pouziva counter, nie `pk`. To s tym `pk` je zbytocne komplikovane.

Reply:
`nr` :ok: